src/Cache/CacheInterface.php

```diff
@@ -17,6 +17,8 @@
 /**
  * @author Andreas Möller <am@localheinz.com>
  *
+ * @readonly
```

Comment:
What is the meaning of `readonly` interface?

Reply:
I was wondering if this enforce that whoever implementing the interface would need to apply this too (otherwise some SCA fails).

not sure on it tho. open for both

Reply:
as we cannot declare readonly interfaces (contrary to `readonly class Foo {}`, worth to revert the change for interfaces 👍🏻

Reply:
👍 also because interfaces cannot contain properties

Reply:
@mvorisek [they can in PHP 8.4](https://www.php.net/manual/en/language.oop5.interfaces.php#language.oop5.interfaces.properties), but this has nothing to do with `readonly` keyword anyway.